graphql_compiler/compiler/ir_lowering_match.py

```diff
@@ -390,6 +390,53 @@ def remove_backtrack_blocks_from_fold(folded_ir_blocks):
     return new_folded_ir_blocks
 
 
+def filter_local_field_existence(field_name):
```

Comment:
since all four of these functions are just private (file-local) helper functions, make their names start with underscores